### ISSUE TITLE
please provide a pkgconfig

### ISSUE THREAD

**Issue by alinelena** [zerothi/fdict#21]
Please provide a package config as part of the installation process.

here is a simple template

```
prefix=@prefix@
exec_prefix=${prefix}
bindir=${exec_prefix}/bin
includedir=${prefix}/include
libdir=${exec_prefix}/lib

Name: fdict
Description: Fortran type-free variables and type-free dictionaries
Version: 0.6.0
URL: https://github.com/zerothi/fdict
Cflags: -I${includedir}
Libs: -L${libdir} -lfdict


```

**Comment by zerothi** [comment]
Thanks, I am on it.

**Comment by zerothi** [comment]
Fixed in cb6c94b18c2c930acebb3007022b4fd8bfdfe87a
